views/index.ejs

```diff
@@ -9,7 +9,7 @@
     <p>Please log in to start messaging</p>
     <form action="/login" method="POST">
       E-Mail: <input type="text" name="email" size="25" /><br /><br />
-      Password: <input type="password" name="password" size="25" /><br /><br />
+      Magic word: <input type="password" name="password" size="25" /><br /><br />
```

Comment:
I agree that "magic word" is much better than "password". But can you also capitalize the "W"?